packages/core/models/input.model.ts

```diff
@@ -0,0 +1,14 @@
+import { HTMLProps } from 'react'
+
+export interface IInput extends HTMLProps<HTMLInputElement> {
+  /** String */
+  label?: string
+  /** String */
+  name: string
+  /** String */
+  style?: object
+  /** String */
+  className?: string
+  /** String */
+  error?: string
+}
```

Comment:
Somethings as `className?` is specific to React Component, move it (you can use Button as example).